### DISCUSSION TITLE
Simplifying Grasping Process in Webots Simulation

### DISCUSSION THREAD

**Discussion by niklasconbotics** [cyberbotics/webots#6331] [General]
Hello everyone,

I am currently working on a project where the main focus is not on the grasping mechanism, but I still need to ensure a reliable method for this action within the Webots simulation environment.

My question to the community is, are there any ways to simplify the grasping process in Webots? Perhaps some sort of an existing library or module I could utilize? Or maybe a certain technique or approach that would help streamline this part of my project?

Please note that I am not looking for a highly detailed or complex grasping mechanism. Instead, I need a straightforward, reliable solution that would work efficiently without taking much of the project's time and resources. Any guidance or suggestions would be greatly appreciated.

Thank you for your time and assistance.

Best Regards,
Niklas

**Comment by omichel** [comment]
You should probably use the brand new [VaccumGripper](https://cyberbotics.com/doc/reference/vacuumgripper) device that was introduced in Webots R2023b. See the [demo world](https://github.com/cyberbotics/webots/blob/master/projects/samples/devices/worlds/vacuum_gripper.wbt).
